plugin.yaml

```diff
@@ -295,6 +295,9 @@ relationships:
 
   cloudify.openstack.server_connected_to_port:
     derived_from: cloudify.relationships.connected_to
+    source_interfaces:
```

Comment:
this looks really weird now, since there's no 'attach'.
the problem isnt your new code though, but rather the way we connect server to networks. the server module should also support dynamic linking to a network/port and then this relationship would have the opposite interface operation as well.